user_docs/en/changes.t2t

```diff
@@ -45,7 +45,7 @@ There are many minor bug fixes for applications, such as Thunderbird, Adobe Read
 - Braille:
   - Added support for the BrailleEdgeS2, BrailleEdgeS3 braille device. (#16033, #16279, @EdKweon)

```

Comment:
Didn't change it entirely here, but consider whether these are displays or devices. Also, I am assuming there are two of them from what is written, not one device with two identifiers. (@EdKweon?)

```suggestion
  - Added support for the BrailleEdgeS2 and BrailleEdgeS3 braille devices. (#16033, #16279, @EdKweon)
```

Reply:
```suggestion
  - Added support for the BrailleEdgeS2 and BrailleEdgeS3 displays. (#16033, #16279, @EdKweon)
```